src/components/CopyUrlButton/docs/CopyUrlButtonWiringExample.st.css

```diff
@@ -0,0 +1,7 @@
+.root {
```

Comment:
We can remove this file if it's not needed

Reply:
it's used in story (`import * as CopyUrlButtonWiringExampleCSSRaw from '!raw-loader!./CopyUrlButtonWiringExample.st.css';`)